### ISSUE TITLE
Custom keyboard key for RE4 Tweaks menu?

### ISSUE THREAD

**Issue by Mister-Curious** [nipkownix/re4_tweaks#82]
As discussed in a [previous thread](https://github.com/nipkownix/re4_tweaks/issues/29#issuecomment-1006256993), we came to the understanding that **Home + End** wasn't the best key combo for toggling the Debug as many users don't have keyboards with those keys. Now another user has brought it to my attention that **F1** is actually used by Raz0r's trainer, as well as some other custom mods people have made. 

Would it be possible to include  an option to use a custom keyboard key instead of F1 in the [MISC] section like what was done with:
 `DebugMenuKeyCombination = CTRL+F3` ?
 maybe something like:
  `RE4TweaksMenuKeyCombination = CTRL+F1` ?
  
You could still default to F1 , but give a few examples in the ini for people to draw from for formatting purposes. 
 
 Thanks!

**Comment by emoose** [comment]
Hmm, maybe should add a `[HOTKEYS]` section to INI & allow customizing them all in there, instead of having `XXKeyCombination` settings in different categories. 

Something like
```INI
[HOTKEYS]
; Key combinations for various RE4 tweaks features
; All keys can be combined (requiring multiple to be pressed at the same time) by using + symbol between key names
; (see top of Settings.cpp file for possible key names to use)

; Key-combination to open the re4_tweaks config UI
ConfigMenu = F1

; Key-combination to open the re4_tweaks debug console (only in certain re4_tweaks builds)
Console = F2

; Key-combination to make the "tool menu" debug menu appear
; (Requires EnableDebugMenu = True above)
DebugMenu = CTRL+F3
```

**Comment by nipkownix** [comment]
The new section could be a good idea.
Could move the inventory flipping and qte bindings there too, I guess.

(@emoose just sent you another email, in albert's behalf this time :p)

**Comment by emoose** [comment]
@nipkownix replied, also forgot to mention in that email, but can you let albert know I'm thankful for the offer too 😄

**Comment by nipkownix** [comment]
No worries, I'll let him know right away!

**Comment by Mister-Curious** [comment]
thanks for the speedy reply and willingness to implement this.

**Comment by nipkownix** [comment]
Implemented as of 5d9fe7fae761786b.
Here's a build:

[re4_tweaks-hotkeys.zip](https://github.com/nipkownix/re4_tweaks/files/8009675/re4_tweaks-hotkeys.zip)

No support for mouse keys for now though. Have yet to decide what to do with those.

Ended up making some big changes, so hopefully nothing breaks. Let me know if it is all good.
Thanks!
